.ksql/GRB_0/GRB_01_STREETNAME_SNAPSHOT_OSLO_STREAM_FLATTEN.ksql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE STREAM IF NOT EXISTS STREETNAME_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB
+WITH (KAFKA_TOPIC='stg.streetname.snapshot.oslo.flatten.grb', PARTITIONS=1, VALUE_FORMAT='JSON_SR', KEY_FORMAT='JSON_SR')
+AS SELECT
+  CAST(IDENTIFICATOR->OBJECTID AS INT) Id,
+  CAST(AS_VALUE(IDENTIFICATOR->OBJECTID) AS INT) StrnmId,
+  IFNULL(FILTER(STRAATNAMEN, (X) => (X->TAAL = 'nl'))[1]->SPELLING,
+    IFNULL(FILTER(STRAATNAMEN, (X) => (X->TAAL = 'fr'))[1]->SPELLING,
+      FILTER(STRAATNAMEN, (X) => (X->TAAL = 'de'))[1]->SPELLING)) Strnm,
+  GEMEENTE->OBJECTID NisCode,
+  CASE WHEN IDENTIFICATOR->ID is null THEN TRUE ELSE FALSE END Removed
+FROM STREETNAME_SNAPSHOT_OSLO_STREAM
+PARTITION BY CAST(IDENTIFICATOR->OBJECTID AS INT);
```

Comment:
Shouldn't we be using the message key to partition by and add it as the objectid? 
![image](https://user-images.githubusercontent.com/18366653/225835872-a19ec9b4-6759-4865-a350-3a12e23ca8c5.png)

Reply:
No, because GRB uses Objectid as PK

Reply:
They indeed use that as a column name, but we put the MessageKey value in there, see commit of @ArneD https://github.com/Informatievlaanderen/streetname-registry/commit/eada7f4d4733f33e3b2e910b23157d5b00af0b02

Reply:
Fixed